type: code
files:
- name: main.py
  visible: true
  learner_created: false
feedback_link: https://hyperskill.org/learn/step/8443#comment
status: Failed
feedback:
  message: |
    Failed test #1 of 25. Runtime error

    This is a sample test from the problem statement!

    Test input:
    3.987
    -10
    Correct output:
    IV

    Your code output:


    Error:
    Traceback (most recent call last):
      File "jailed_code", line 1, in <module>
        number1 = int(input())
    ValueError: invalid literal for int() with base 10: '3.987'
  time: Wed, 16 Dec 2020 07:14:07 UTC
record: -1
